docker/Dockerfile

```diff
@@ -56,10 +57,10 @@ RUN set -ex \
  #     gpg --keyserver "$server" --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 && break || : ; \
  #   done \
  && curl -o /usr/local/bin/gosu -SL "https://github.com/hysds/gosu/releases/download/${GOSU_VERSION}/gosu-amd64" \
- && curl -o /usr/local/bin/gosu.asc -SL "https://github.com/hysds/gosu/releases/download/${GOSU_VERSION}/gosu-amd64.asc" \
- #&& gpg --verify /usr/local/bin/gosu.asc \
- && rm /usr/local/bin/gosu.asc \
- #&& rm -r /root/.gnupg/ \
+#RUN curl -o /usr/local/bin/gosu.asc -SL "https://github.com/hysds/gosu/releases/download/${GOSU_VERSION}/gosu-amd64.asc"
+#RUN gpg --verify /usr/local/bin/gosu.asc
+#RUN rm /usr/local/bin/gosu.asc
+#RUN rm -r /root/.gnupg/
```

Comment:
@pymonger @hookhua What is the history of this? We were able to build the containers fine without this, but not sure if long term, we should retain this.

Reply:
@MJJoyce Just talked with @pymonger about this. Feel free to merge in. Could you put a comment on this part that says that says something to the affect of we're leaving this commented out for the time being in case we need to put it back? I think the intent was to adhere to best practices by doing pgp verification, but you're right, it doesn't hurt anything if it's left out. The subsequent commands to make gosu executable that follow this block would fail the build if we had issues downloading gosu in the first place.